src/test/resources/dialogmelding_dialog_svar_innkalling_dialogmote.xml

```diff
@@ -1,77 +1,76 @@
+                    </TeleCom>
+                    <TeleCom>
+                        <TypeTelecom DN="Fax" V="F"/>
+                        <TeleAddress V="fax:87 65 43 21"/>
+                    </TeleCom>
                     <HealthcareProfessional>
                         <FamilyName>Valda</FamilyName>
                         <MiddleName>Fos</MiddleName>
                         <GivenName>Inga</GivenName>
-                        <Ident>
-                            <Id>1234</Id>
-                            <TypeId DN="Identifikator fra Helsetjenesteenhetsregisteret" S="1.16.578.1.12.4.3.1.8116" V="HER" />
-                        </Ident>
                         <Ident>
                             <Id>1234567</Id>
-                            <TypeId DN="HPR-nummer" S="1.16.578.1.12.3.1.1.8116" V="HPR" />
-                        </Ident>
-                        <Ident>
-                            <Id>04030350265</Id>
-                            <TypeId DN="F�dselsnummer" V="FNR" S="6.87.654.3.21.9.8.7.6543.2198" />
+                            <TypeId DN="HPR-nummer" S="2.16.578.1.12.4.1.1.8116" V="HPR"/>
                         </Ident>
                         <Ident>
-                            <Id>45069800525</Id>
-                            <TypeId DN="D-nummer" V="DNR" S="6.16.578.1.12.4.1.4.8116" />
+                            <Id>1234</Id>
+                            <TypeId DN="HER-id" S="2.16.578.1.12.4.1.1.8116" V="HER"/>
                         </Ident>
-                        <Address>
-                            <StreetAdr>Oppdiktet gate 9001</StreetAdr>
-                            <PostalCode>9998</PostalCode>
-                            <City>Oppdiktet</City>
-                        </Address>
-                        <TeleCom>
-                            <TeleAddress V="98765432" />
-                            <TypeTelecom DN="Hovedtelefon" />
-                        </TeleCom>
                     </HealthcareProfessional>
                 </Organisation>
             </Sender>
             <Receiver>
                 <Organisation>
                     <OrganisationName>NAV</OrganisationName>
                     <Ident>
-                        <Id>889640782</Id>
-                        <TypeId DN="Organisasjonsnummeret i Enhetsregister (Br�nn�ysund)" S="2.16.578.1.12.4.1.1.9051" V="ENH" />
+                        <Id>79768</Id>
+                        <TypeId DN="Identifikator fra Helsetjenesteenhetsregisteret (HER-id)" S="2.16.578.1.12.4.1.1.9051" V="HER"/>
                     </Ident>
                     <Ident>
-                        <Id>79768</Id>
-                        <TypeId DN="Identifikator fra Helsetjenesteenhetsregisteret (HER-id)" S="2.16.578.1.12.4.1.1.9051" V="HER" />
+                        <Id>889640782</Id>
+                        <TypeId DN="Organisasjonsnummeret i Enhetsregisteret" S="2.16.578.1.12.4.1.1.9051" V="ENH"/>
```

Comment:
Det ser jo veldig ut til at EPJ-ene har bytta ut orgnr med Herid, også for NAV..